auth/src/shared/interfaces/request-with-user.dto.ts

```diff
@@ -0,0 +1,5 @@
+import { JWTPayload } from '../dtos/jwt-payload.dto';
+
+export interface RequestWithUser extends Request {
```

Comment:
This one extends Request but the request with refresh jwt doesn't.
It isn't necessarily needed but it's a 'Request with ...' which might be deceiving in the refresh token interface

Reply:
Unified